doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst

```diff
@@ -0,0 +1,172 @@
+Sagemaker Distributed Data Parallel 1.2.1 Release Notes
+=======================================================
+
+*Date: June. 29. 2021*
+
+**New Features:**
+
+-  Added support for TensorFlow 2.5.0.
+
+**Improvements**
+
+-  Improved performance on a single node.
+-  Improved performance on small clusters (2-4 nodes).
+
+**Bug fixes**
+
+-  Device selection for SageMaker.
```

Comment:
@yselivonchyk do we need to include it also? it is also very technical

Reply:
I agree, we probably don't need to share that with the user

Reply:
removing the list item